### DISCUSSION TITLE
Have you explored the new Mobile App Development Experience yet?

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#60888] [GitHub Education]
### Hi everyone,

![Mobile App Development Card](https://github.com/community/community/assets/107881423/8479f216-c4d4-4d88-a74d-b7581ccb3740)

Have an idea for a mobile app you'd like to bring to life? This [Mobile Development Experience](https://education.github.com/experiences/mobile_app_development) gives you everything you need to design, build, test and deploy on mobile. This Experience features pack partner offers from Bump.sh, Replit, Kodika, FrontendMasters, LambdaTest, LingoHub and Azure. 

**Start off on the right foot**
Kick off your project with a mobile template in Codespaces or browse through the courses on offer from FrontendMasters where you can learn how to build responsive applications using Flutter, Swift, Kotlin and more.

**Build a solid toolkit**
Explore pack offers from Bump, Replit, Kodika and Azure to build, document and deploy your code. Level up by making sure your app looks and functions as it should, and is as accessible as possible, with the help of LambdaTest and LingoHub.

**Leverage Open Source resources**
Don’t underestimate the power of open source when building your mobile app. Explore free resources from NativeScript and Flutter to make your mobile apps a reality.

To start, become a verified student on [Global Campus](https://education.github.com/pack/join) and start redeeming your benefits.

**Comment by Xelliese** [comment]
I've also been checking out that Mobile App Development Experience. It looks like it has everything you need to get started or improve your app building skills. I’ve found it hard before to find the right help for my own app ideas, so this setup might be just what I need.

I also needed some custom help because traditional app development was too slow for me. Discovering low-code and no-code platforms really changed the game. They cut down both time and cost. I ended up working with a team that uses platforms like Bubble and Airtable, focusing on [custom application development](https://www.crispysoftwaresolutions.com/). This approach was faster and more tailored to my needs, making everything easier and more personalized.

**Comment by nothatkidreal** [comment]
Kodika’s offer no longer works and has been reported as “not working” as of around when you guys announced this. Potentially get in contact with them?
